pkg/mapping/processors/assetinventory.go

```diff
@@ -241,22 +242,24 @@ func mergeAnnotations(annos1, annos2 *structpb.Struct) (*structpb.Struct, error)
 	return s, nil
 }
 
-// parseProject gets "Project" from "ResourceName" follows format here:https://cloud.google.com/asset-inventory/docs/resource-name-format.
-// Return empty string for resources such as GCS bucket won't include "Project" info in its "ResourceName".
-func parseProject(resourceName string) (string, error) {
+// parseScope gets "project/folder/orgnization" from "ResourceName" follows format here:https://cloud.google.com/asset-inventory/docs/resource-name-format.
+// Return empty string for resources such as GCS bucket won't include "project/folder/orgnization" info in its "ResourceName".
+func parseScope(resourceName string) (string, error) {
 	s := strings.Split(resourceName, "/")
-	project := ""
+	scopePrefix := ""
+	scope := ""
 	for i, e := range s {
-		if e == "projects" {
+		if _, ok := server.SupportedResourceScope[e]; ok {
```

Comment:
This is bad. Don't depend on `server` package. Repeating projects/folders/organizations is ok